pkg/util/objectpool/objectpool.go

```diff
@@ -32,53 +32,51 @@ type PoolObject interface {
 }
 
 type (
+	// MultiPool manage multi Pool.
+	MultiPool struct {
```

Comment:
`Pool` is designed to be a library or module, but the `MultiPool` is more tightly bound to the application using the `Pool`, so it should be put into the application code instead of here.

Reply:
I will try